packages/p2p/src/components/modal-manager/modals/activate-deactivate-error-modal/activate-deactivate-error-modal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { generateErrorDialogTitle } from 'Utils/adverts';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+
+const ActivateDeactivateErrorModal = () => {
+    const { my_ads_store } = useStores();
+    const { hideModal, is_modal_open } = useModalManagerContext();
+
+    React.useEffect(() => {
+        return () => {
+            my_ads_store.setActivateDeactivateErrorMessage('');
+        };
+    }, []);
+
+    return (
+        <Modal
+            className='p2p-my-ads__modal-error'
+            has_close_icon={false}
+            is_open={is_modal_open && Boolean(my_ads_store.activate_deactivate_error_message)}
+            small
+            title={generateErrorDialogTitle(my_ads_store.error_code)}
+        >
+            <Modal.Body>
+                <Text as='p' size='xs' color='prominent'>
+                    {my_ads_store.activate_deactivate_error_message}
+                </Text>
+            </Modal.Body>
+            <Modal.Footer>
+                <Button has_effect large onClick={hideModal} primary text={localize('Ok')} />
+            </Modal.Footer>
+        </Modal>
+    );
+};
+
```

Comment:
Added 2 new modals: `ActivateDeactivateErrorModal` and `QuickAddErrorModal`

Reply:
can we use `ErrorModal` instead?

Reply:
Oh yaaa I could have used that, I'll refactor it to use `ErrorModal` instead